### PR TITLE
Adding info on model summary tools to Deep Dive 1.

### PR DESCRIPTION
**Pull request by jimn2**
Kartik, when you and I talked, we discussed adding information about model summary tools to DD1.  This is my first cut at doing that.

### CODE REVIEW

File: benchmarks/templates/benchmarks/tutorials/models/deepdive_1.html [hunk 1]
```diff
@@ -263,6 +263,73 @@ <h3 class="benefits_heading is-size-3-mobile">Part 8: Putting it All Together </
...
+
+<div class="box leaderboard-table-component">
+    <h3 class="benefits_heading is-size-3-mobile">Part 9: Model Summary Tools</h3>
+    <p class="benefits_info is-size-5-mobile shorter">
+        Depending on how a model is created, you can use one of the following tools to get summary information about it:
+    </p>
+    <ul class="benefits_list">
+        <li><strong>Keras:</strong> If a model is created with Keras, the built-in <span class="special_format">model.summary()</span>
+            method can be used to display a concise summary of the model, showing the configuration of each layer,
+            including the name, type, output shape, and parameter count.</li>
```
Comment (KartikP) [inline benchmarks/templates/benchmarks/tutorials/models/deepdive_1.html:275]:
```suggestion
```

Brain-Score 2.1 (i.e., 3.11 conversion) will only support PyTorch models, so this part might be obsolete. Could theoretically still be useful to include?

File: benchmarks/templates/benchmarks/tutorials/models/deepdive_1.html [hunk 2]
```diff
@@ -263,6 +263,73 @@ <h3 class="benefits_heading is-size-3-mobile">Part 8: Putting it All Together </
...
+    </ul>
+    <p class="benefits_info is-size-5-mobile shorter">
+        Here’s how to set up and use a customizable script in PyTorch to summarize different models by specifying a model identifier (you can
+        also embed these commands in your own model or other code):
+    </p>
+    <ol class="benefits_list">
+        <li>Ensure your environment is set up as per the <a href="https://www.brain-score.org/tutorials/models/quickstart">Quickstart Tutorial</a>.</li>
+        <li>Create a new Python file named <span class="special_format">modelsummary.py</span> in the <span class="special_format">vision</span> directory.</li>
+        <li>Copy and paste the following script into the file:</li>
+    </ol>
```
Comment (KartikP) [inline benchmarks/templates/benchmarks/tutorials/models/deepdive_1.html:289]: Is there a way to fix the indenting of this list? Same with point 4 and 5.

The image below is how the list renders for me locally.

image

Comment (jimn2) [reply benchmarks/templates/benchmarks/tutorials/models/deepdive_1.html]: Removed reference to Keras and fixed numbering indentation.

**Review by KartikP — APPROVED** [review]
LGTM

**Pull request merged**